oom_kill/README.md

```diff
@@ -17,11 +17,14 @@ On Debian-like distributions, install the kernel headers like this:
 apt install -y linux-headers-$(uname -r)
 ```
 
-On RHEL-like distributions, install the kernel headers like this:
+On RHEL-like distributions, install the kernel headers and kernel devels like this:
```

Comment:
In fact, the `kernel-devel` package also contains kernel headers.
The difference is that `kernel-headers` contains the headers for the interface between the kernel and the userspace whereas `kernel-devel` contains the headers for the interface between the kernel and the kernel modules.

Here are the descriptions of the two packages:
```
Name        : kernel-headers
Description : Kernel-headers includes the C header files that specify the interface
            : between the Linux kernel and userspace libraries and programs.  The
            : header files define structures and constants that are needed for
            : building most standard programs and are also needed for rebuilding the
            : glibc package.

Name        : kernel-devel
Description : This package provides kernel headers and makefiles sufficient to build modules
            : against the kernel package.
```

So, for me, it would be OK to let it as is:
```suggestion
On RHEL-like distributions, install the kernel headers like this:
```

or, if we want to highlight the fact that there are two distinct packages to install:
```suggestion
On RHEL-like distributions, install the kernel headers and the kernel development package like this:
```